Trannet/Program.cs

```diff
@@ -18,6 +15,109 @@
 
 app.UseAuthorization();
 
-app.MapControllers();
+var (Trips, TripsIxByRoute) = LoadTrips();
+var (StopTimes, StopTimesIxByTrip) = LoadStopTimes();
+
+app.MapGet("/schedules/{routeId}", (string routeId) =>
+    {
+        var trips = new List<TripResponse>();
+        if (TripsIxByRoute.ContainsKey(routeId))
+        {
+            var tripIxs = TripsIxByRoute[routeId];

```

Comment:
```suggestion
        if (TripsIxByRoute.TryGetValue(routeId, out var tripIxs))
        {
```

Reply:
oups, thanks! indeed, I forgot to update this code while I did it on a separate iteration 😓